### ISSUE TITLE
Make 


### ISSUE THREAD

**Issue by 09306840723** [09306840723/matplolib#2]
import numpy as np
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler
from sklearn.metrics import accuracy_score
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

# بارگذاری داده‌ها
data = pd.read_csv('breast_cancer_data.csv')

# جدا کردن ویژگی‌ها و برچسب‌ها
X = data.drop(columns=['diagnosis'])
y = data['diagnosis'].apply(lambda x: 1 if x == 'M' else 0)  # M برای بدخیم و B برای خوش‌خیم

# تقسیم داده‌ها به مجموعه‌های آموزشی و آزمایشی
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# استانداردسازی داده‌ها
scaler = StandardScaler()
X_train = scaler.fit_transform(X_train)
X_test = scaler.transform(X_test)

# ساخت مدل شبکه عصبی
model = Sequential()
model.add(Dense(16, input_dim=X_train.shape[1], activation='relu'))
model.add(Dense(8, activation='relu'))
model.add(Dense(1, activation='sigmoid'))

# کامپایل کردن مدل
model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])

# آموزش مدل
model.fit(X_train, y_train, epochs=50, batch_size=10, verbose=1)

# ارزیابی مدل
y_pred = (model.predict(X_test) > 0.5).astype("int32")
accuracy = accuracy_score(y_test, y_pred)
print(f'Accuracy: {accuracy * 100:.2f}%')
